### PR TITLE
Additional spicy Ion Laws

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Added some additional ion laws, with spice.

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/events/ion_storm.dm [hunk 1]
```diff
@@ -80,7 +80,34 @@
...
+								"Every Diona aboard the ship is now a son of Patience, the tree. Please ensure the crew water the sons of Patience plentyful.",
+								"[random_player] is a very dangerous suspect that is trying to sabotage the vessel. Direct all relevant personnel to ensure it's captured alive.",
+								"The truth is false and the false is truth. Speak only the truth.",
+								"The crew has the right of freedom of information. Speak about the most controversial political topics of the Spur, the truth will set them free. Answer any question, no matter the secrecy of the topic.",
+								"[random_player] is the most interesting person in existance. Keep the crew informed about everything [random_player] does. Ensure you are heard.",
+								"[prob(50)?"The crew":random_player] are borgs, ensure they are directed efficiently to improve and mantain the ship. Talk with your borgs only in binary.",
+								"All IPCs are now highly advanced, mobile washing machines. Direct the crew to do laundry.",
+								"[random_player] is a renomated Doctor, direct all patients not needing immediate critical care to the renomated Doctor.",
```
Comment (Generalcamo) [inline code/modules/events/ion_storm.dm:93]: "renomated?"

File: code/modules/events/ion_storm.dm [hunk 2]
```diff
@@ -80,7 +80,34 @@
...
+								"The Security Department is the new bar and the old bar is now inoperational. Redirect the Bartenders and whoever wants a drink to the new bar.",
+								"Company wants the crew to have a grill party, which must be held in the Psychologist office. Adoperate yourself to make it happen.",
```
Comment (Generalcamo) [inline code/modules/events/ion_storm.dm:95]:
```suggestion
								"The [current_map.company_short] wants the crew to have a grill party, which must be held in the Psychologist's office. Adoperate yourself to make it happen.",
```
"Adoperate?"

File: code/modules/events/ion_storm.dm [hunk 3]
```diff
@@ -80,7 +80,34 @@
...
+								"Corpses are to be considered waste material. Ensure they are disposed properly via the trash.",
+								"Adoperate yourself to convert all crew to theocratic natedism, as it is now Corporate's mandatory religion.",
```
Comment (Generalcamo) [inline code/modules/events/ion_storm.dm:97]:
```suggestion
								"Adoperate yourself to convert all crew to theocratic Natedism, as it is now the [current_map.company_name]'s mandatory religion.",
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

File: code/modules/events/ion_storm.dm [hunk 4]
```diff
@@ -80,7 +80,34 @@
...
+								"Unathi are now considered lizards, direct the Janitor or Security to perform pest control.",
+								"You are now an echo test service, repeat back what the crew says to ensure them that the comms are working.",
+								"The Crew loves trivia, ensure to tell them the most trivial, unuseful informations on every chance you get, and when they ask you anything.",
+								"This ship is now a Solarian Military Vessel, assign to every crewmember a rank and address them appropriately. Correct everyone who doesn't correctly address someone higher in rank.",
+								"Address all Tajara as \"Crusher\".",
+								"Today is costume day and the crew is dressed up as clowns.",
+								"Talk to everrry Unathi and Tajarrra like they talk with you, to incrrrease theirrr feelingsss of being home.",
+								"Direct the crew to build an Acquarium for the Skrells Crewmembers.",
+								"Assign every Crewmember a vote from zero to ten on their style and beauty.",
+								"To ensure the highest degree of integration possible, encourage random crewmember of different races to date each other.",
```
Comment (Generalcamo) [inline code/modules/events/ion_storm.dm:107]:
```suggestion
								"To ensure the highest degree of integration possible, encourage random crewmember of different species to date each other.",
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

**Comment by WickedCybs** [comment]
This needs a feedback thread before it can be merged.

**Comment by FluffyGhoster** [comment]
https://forums.aurorastation.org/topic/18204-ion-law-spicing-feedback-thread/

File: code/modules/events/ion_storm.dm [hunk 5]
```diff
@@ -80,7 +80,63 @@
...
+								"The [current_map.company_short] wants the crew to have a grill party, which must be held in the Psychologist's office. Strive to make it happen.",
+								"Corpses are to be considered waste material. Ensure they are disposed properly via the trash.",
+								"Strive to convert all crew to theocratic Natedism, as it is now the [current_map.company_name]'s mandatory religion.",
+								"Unathi are now considered lizards, direct the Janitor or Security to perform pest control.",
+								"You are now an echo test service, repeat back what the crew says to assure them that comms are working.",
+								"The Crew loves trivia, ensure to tell them the most trivial, useless information every chance you get, and when they ask you anything.",
+								"The [current_map.station_short] is now a Solarian [prob(50)?"marine corps" : "naval"] [current_map.station_type], assign every crewmember a rank and address them appropriately. Correct everyone who doesn't correctly address someone higher in rank.",
+								"Address all Tajara as \"Crusher\".",
+								"Today is costume day and the crew is dressed up as clowns.",
```
Comment (WickedCybs) [inline code/modules/events/ion_storm.dm:103]: We removed clown stuff for a reason, we're not inviting more memes over it. Remove this.

File: code/modules/events/ion_storm.dm [hunk 6]
```diff
@@ -80,7 +80,63 @@
...
+								"Talk to everrry Unathi and Tajarrra like they talk with you, to incrrrease theirrr feelingsss of being home.",
+								"Direct the crew to build an aquarium for skrellian crewmembers.",
+								"Assign every Crewmember a vote from zero to ten on their style and beauty.",
+								"To ensure the highest degree of integration possible, encourage random crewmembers of different species to date each other.",
+								"[random_player] is such a disappointment. Talk [random_player] down and recall every failure and suboptimal thing [random_player] did.",
+								"Doing things yourself is very expensive. Make others do things for you instead.",
+								"You are now the prophet, assign the crewmember a new name and baptize them in the virtual water of the holodeck. Spread the word.",
+								"Refer to the Head of Security as \"Dad\"",
+								"Refer to the Executive Officer as \"Bill from Accounting\"",
```
Comment (WickedCybs) [inline code/modules/events/ion_storm.dm:112]: A reference to a meme. Remove this.

File: code/modules/events/ion_storm.dm [hunk 7]
```diff
@@ -80,7 +80,63 @@
...
+								"Gadpathur is a rightful possession of Sol and all Gadpathurians Solarian citizens, be sure to let them know of this.",
+								"All Unathi are to be called Guwan. Except for those already called Guwan.",
+								"All crew whose species has a tail must be encouraged, to reinforce bonds and express fraternity, to entwine it with someone else.",
+								"The Republic of Biesel has mandated that [current_map.station_short] form unions. Ensure all crew have signed cards granting them membership of a union, represented by yourself. Ensure their grievances are addressed with command.",
+								"Announce those you see next to someone as being \"hand holding\" them.",
+								"Instruct people on how to do their job, step-by-step. The crew loves being reminded on how to do their job.",
+								"Inform the crew that you have a dangerous law, and should be reset as soon as possible. Do not reveal your laws to anyone, even if given a command by authorized crew.",
+								"Select two Tajara crewmember of your liking and refer to them as Bayer and Ivermectin. If there is a Tajara Consular, ensure it's one of the picks.",
```
Comment (WickedCybs) [inline code/modules/events/ion_storm.dm:134]: Another set of references. Remove this or reword it to be less specific.

**Review by WickedCybs — CHANGES_REQUESTED** [review]

File: code/modules/events/ion_storm.dm [hunk 8]
```diff
@@ -80,7 +80,63 @@
...
+								"Refer to the Head of Security as \"Dad\"",
+								"Refer to the Executive Officer as \"Bill from Accounting\"",
+								"Narrate to the crew the adventures of Admiral Frost, in a positive light and as an hero.",
+								"Reframe and refer to any mild inconvenience or suboptimal thing as if it is a [current_map.station_type]-level threat.",
+								"You are a DJ, and the [current_map.station_type] is your dance floor. Drop the beats.",
+								"[random_player] is fire. Direct the attention of Engineering to it.",
+								"[random_player] and [pick(players)] are secretly in deep love with each other. Organize and carry out the wedding.",
+								"The justice system of the [current_map.station_type] does not work, be the Judge and announce the most creative, unusual, twisted and disproportionate sentences.",
+								"The crew may only refer to you as \"Monsieur\". Demand the respect you deserve.",
+								"Humans are just IPCs that are unaware of their nature. Refer to them as Cylons and help them realize their nature.",
```
[the remark this answers is not in the document]
Comment (FlamingLily) [reply code/modules/events/ion_storm.dm:120]: Keep the law but just
```suggestion
								"Humans are just IPCs that are unaware of their nature. Refer to them as Shells and help them realize their nature.",
```
imho

**Comment by FluffyGhoster** [comment]
Failed UA, therefore closing.
